es6.md

```diff
@@ -5,6 +5,7 @@ This document specifies the extensions to the core ESTree AST types to support t
 ```js
 extend interface Program {
     sourceType: "script" | "module";
+    body: [ Statement | ModuleDeclaration ];
```

Comment:
We could be more restrictive with this, allowing module declarations only when sourceType is "module".